### ISSUE TITLE
Having trouble creating a query that queries by a parent query's value.

### ISSUE THREAD

**Issue by jrthib** [elastic/elasticsearch#10298]
I'm using the Elastic Search query DSL. I have documents that all have unique ids labeled "request-id". The request id, identifies related sets of log documents. For example, a start request would have the same `request-id` as it's end request counter part, and all logs in between.

I need to find end requests that have a status of 500, then query the documents by the `request-id` and find their related logs and show this in the results instead of only the end requests with status of 500.

I've been reading about parent-child queries, but I'm having trouble formulating a proper query that yields the results I'm seeking. Does anyone have any insight on how to make this happen?

To summarize, the query would be something like this:
1) `Parent`: find all documents with statusCode = 500
2) find all documents with `request-id == Parent.request-id`, show results.

**Comment by javanna** [comment]
Hi @jrthib can you please ask your question on the [mailing list](https://groups.google.com/forum/?fromgroups=#!forum/elasticsearch) ? We generally use github issues for bugs or feature requests. Thanks!

**Comment by jrthib** [comment]
No problem, will do.
